src/shim/go.mod

```diff
@@ -22,7 +22,7 @@ require (
 require (
 	github.com/Microsoft/go-winio v0.5.2 // indirect
 	github.com/Microsoft/hcsshim v0.9.3 // indirect
-	github.com/cilium/ebpf v0.9.1 // indirect
+	github.com/cilium/ebpf v0.11.0 // indirect
```

Comment:
this one can be dropped.

Reply:
I may have misunderstood you. I tried dropping that line and CI choked...
``` 
bin/containerd-shim-rune-v2
go: inconsistent vendoring in /home/ecc/actions-runner/_work/enclave-cc/enclave-cc/src/github.com/confidential-containers/enclave-cc/src/shim:
	github.com/cilium/ebpf@v0.[11](https://github.com/confidential-containers/enclave-cc/actions/runs/6226114935/job/16897952955?pr=222#step:8:12).0: is marked as explicit in vendor/modules.txt, but not explicitly required in go.mod
```

Reply:
I meant that the modification can be dropped and we keep v0.9.1 in go.mod. It's good to `go mod tidy && go mod vendor` after that.